src/config.ts

```diff
@@ -4,9 +4,13 @@ export default {
   preventWords: [
     'guyz',
     'guyzz',
+    "dude",
     'bruh',
     'duude',
     'women',
+    'fello',
+    'bro',
+    'guys',
```

Comment:
`bro`, `dude` and `guys` are already flagged by AlexJS library we use http://alexjs.com

I think `fello` is a good addition 👍